web-client/src/modules/create/containers/CreatePostContainer.tsx

```diff
@@ -1,336 +1,152 @@
-        type: requestInfo ? requestInfo.type : DELIVERIES,
-        other: requestInfo ? requestInfo.other : '',
-        description: requestInfo ? requestInfo.description : '',
-      };
-
-      return (
-        <RequestDetails>
-          <NewPost
-            onSubmit={newRequestSubmitHandler}
-            request={request}
-            setStreetAddress={setMapAddress}
-            setMapAddress={() => setStartGeocode(true)}
-            setMyLocation={() => setStartLocateMe(true)}
-          />
-        </RequestDetails>
-      );
-    }
+      pinUserRef: profileState.userRef!,
+      pinUserSnapshot: profileState.profile!.toObject() as IUser,
+      streetAddress: `${address1} ${address2} ${city} ${state} ${postalCode} ${country}`,
+      latLng: new firestore.GeoPoint(coords.latitude, coords.longitude),
+    };
+    dispatch(setRequest(newPost as IRequest, undefined, phoneNumber));
   };
 
-  const maybeRequestReview = () => {
-    if (showReviewPage && requestInfo) {
-      return (
-        <RequestDetails>
-          <PostReview
-            request={requestInfo}
-            saveRequest={() => {
-              reviewRequestSubmitHandler(requestInfo);
-            }}
-            isSubmitting={isSubmitting}
-            goBack={onGoBack}
-          />
-        </RequestDetails>
-      );
-    }
-  };
-
-  const maybeRequestConfirmation = () => {
-    if (showConfirmationPage) {
-      return (
-        <PostConfirmation
-          showModal={showConfirmationPage}
-          closeModal={() => {
-            setShowConfirmationPage(false);
-            // because I could observe race conditions in cloud function
-            setTimeout(() => {
-              history.replace(MyRequestPostsLocationUrl);
-            }, 150);
-            dispatch(resetSetRequestState());
-          }}
+  const postCreationSteps = [
+    {
+      title: 'Details',
+      component: (
+        <PostDetails
+          nextHandler={moveForwards}
+          prevHandler={cancelCreate}
+          postDetails={postDetails}
+          setPostDetails={setPostDetails}
         />
-      );
-    }
-  };
-
-  const instructions = [
-    t('information_modal.NewRequestsContainer.0'),
-    t('information_modal.NewRequestsContainer.1'),
-    t('information_modal.NewRequestsContainer.2'),
-    t('information_modal.NewRequestsContainer.3'),
-    t('information_modal.NewRequestsContainer.4'),
+      ),
+    },
+    {
+      title: 'Map',
+      component: (
+        <PostMap
+          nextHandler={moveForwards}
+          prevHandler={moveBackwards}
+          postDetails={postDetails}
+          addresses={addresses}
+          postLocation={postLocation}
+          setPostLocation={setPostLocation}
+        />
+      ),
+    },
+    {
+      title: 'Summary',
+      component: (
+        <PostSummary
+          prevHandler={moveBackwards}
+          postDetails={postDetails}
+          postLocation={postLocation}
+          submitRequest={submitPost}
```

Comment:
If nit: rename this prop to submitPost since it sounds like this will support requests and offers